documentation/PYTHON.md

```diff
@@ -1,87 +1,150 @@
 # Python Interface {#python_interface}
 
-In the following we will give a detailed overview how to specify models in  Python and how to call the generated simulation files.
+In the following we will give a detailed overview how to specify models in
+Python and how to call the generated simulation files.
 
 ## Model Definition
 
-This guide will guide the user on how to specify models in Python using SBML. For example implementations see the examples in the python/examples directory.
+This guide will guide the user on how to specify models to import and simulate
+them using the Python interface.
+For example implementations see the examples in the `python/examples`
+directory.
 
-### SBML input
+### SBML import
 
-First, import an sbml file using the `amici.sbml_import.SbmlImporter` class:
+[SBML](http://sbml.org/) is a commonly used format for specifying systems
+biology models. To import an SBML model into AMICI, first, load an SBML file
+using the `amici.sbml_import.SbmlImporter` class:
 
     import amici
-    sbmlImporter = amici.SbmlImporter('model_steadystate_scaled.sbml')
-    
-the sbml document as imported by [libSBML](http://sbml.org/Software/libSBML) is available as 
+    sbml_importer = amici.SbmlImporter('model_steadystate_scaled.sbml')
 
-    sbml = sbmlImporter.sbml
+the SBML model as imported by [libSBML](http://sbml.org/Software/libSBML)
+is available as
 
-### Constants 
+    sbml_model = sbml_importer.sbml
 
-parameters that should be considered constants can be specified in a list of strings specifying the respective SbmlId of a parameter.
+#### Constants
 
-    constantParameters=['k4']
+Model parameters that should be considered constants can be specified in a list
```

Comment:
We should ideally say/repeat (or link to a definition) what constants are, i.e. no sensitivity computation w.r.t. those values, they can change for preequi/presim/sim and usually correspond to experimental conditions.

Reply:
Yes. I'd like to include some kind of glossary to not repeat that everywhere.